modules/crypt.cpp

```diff
@@ -501,25 +504,36 @@ class CCryptMod : public CModule {
         }
     }
 
+    void OnNickPrefixCommand(const CString& sCommand) {
+        CString sPrefix = sCommand.Token(1, true);
+
+        if (sPrefix.empty()) {
+            PutModule("Nick Prefix: " + NickPrefix());
+        } else {
+            CString sStatusPrefix = GetUser()->GetStatusPrefix();
+            size_t sp = sStatusPrefix.size();
+            size_t np = sPrefix.size();
+            SetNV(NICK_PREFIX_KEY, sPrefix);
+            PutModule("Setting Nick Prefix to " + sPrefix);
+            if (sStatusPrefix.CaseCmp(sPrefix, std::min(sp, np)) == 0)
+                PutModule("WARNING: overlap with Status Prefix (" + sStatusPrefix + "), this Nick Prefix will not be used!");
+        }
+    }
+
     void OnListKeysCommand(const CString& sCommand) {
-        if (BeginNV() == EndNV()) {
+        if (BeginNV() == EndNV() || BeginNV()->first.Equals(NICK_PREFIX_KEY)) {
```

Comment:
Why do you think it's the first entry? Maybe it's the last entry? Or somewhere in the middle?